docs/requirements.txt

```diff
@@ -5,7 +5,7 @@
 breathe
 ipython
 numpydoc
-pydata-sphinx-theme
+pydata-sphinx-pydata-sphinx-theme==0.14
```

Comment:
```suggestion
pydata-sphinx-theme==0.14
```

Reply:
Whoops, copy paste went wrong .. Thanks for the catch!